src/Debugger.php

```diff
@@ -31,14 +29,12 @@ final class Debugger
 
     /**
      * @param CollectorInterface[] $collectors
-     * @param string[] $ignoredRequests
-     * @param string[] $ignoredCommands
      */
     public function __construct(
         private readonly StorageInterface $storage,
         array $collectors,
-        private readonly array $ignoredRequests = [],
-        private readonly array $ignoredCommands = [],
+        private readonly ?StartupPolicy $startupPolicy = null,
```

Comment:
Why it could null? Let's define it anyway

Reply:
It's require to create new policy classs. We can avoid it. It's easier to use `null`.

Reply:
done